crates/sui-e2e-tests/tests/protocol_version_tests.rs

```diff
@@ -51,6 +51,60 @@ fn test_protocol_overrides_2() {
     );
 }
 
+#[tokio::test]
+async fn test_issue_with_stake_subsidy() {
```

Comment:
I think this is from a commit of mine investigating the issue -- note that this isn't intended to be checked in or landed.